overrides/android-override.json

```diff
@@ -552,6 +552,23 @@
         "clientContentFeatures": {
             "state": "enabled",
             "exceptions": [ ]
+        },
+        "mediaPlaybackRequiresUserGesture": {
+            "state": "enabled",
+            "exceptions": [
+                {
+                    "domain": "clearmemorysolution.com"
+                },
+                {
+                    "domain": "id.onlyfans.com"
+                },
+                {
+                    "domain": "naturemetabolism.com"
+                },
+                {
+                    "domain": "upperbodystretch.com"
```

Comment:
Added 2 domains as we found 2 more website with the same issue. See https://app.asana.com/0/1206374308645191/1206445562784324/f and https://app.asana.com/0/1163321984198618/1205995907661137